docs/docs/import-external-csv-files-to-landing-zone.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Importing external CSV files to the Data Platform Landing Zone
+description: "Overview of how CSV files from external suppliers are imported to the Data Platform Landing Zone"
+layout: playbook_js
+tags: [playbook]
+---
+
+This section covers the technical overview of how external suppliers can import data into the data platform via .csv files.
+
+The terraform script [33-liberator-iam.tf](https://github.com/LBHackney-IT/Data-Platform/tree/main/terraform/33-liberator-iam.tf) provisions an AWS IAM User and associated policies. The policies will allow the external supplier to see an S3 bucket, and upload a file into that bucket and see the file. This can be used by the external provider by executing the following steps:
+
+1. The external supplier logs into AWS using the [AWS Command Line Interface](https://aws.amazon.com/cli/)
+2. The external supplier uses the `aws s3 cp` command with the source location and the destination as arguments, e.g. `aws s3 cp c:\<SUPPLIER_COMPUTER\<SOURCE_FILE_NAME>.csv s3://<DESTINATION_BUCKET>/<LANDING_ZONE_NAME>/`
```

Comment:
Maybe give an example of an S3 path, I'm a little unsure what is meant by "LANDING_ZONE_NAME" here.